### ISSUE TITLE
Are some Document Act Roles not Deontic Roles?

### ISSUE THREAD

**Issue by jonathanvajda** [d-acts/d-acts#34]
Given the definition of deontic roles, it seems to 1:1 overlap entirely with socially normative roles. However, some terms seem not to be child-classes of deontic roles that have normative force. It is a commonly-held view that speech acts are inherently normative. If document acts are analogous to speech acts except that the social act has a document (rather than speech) intervene, then document acts are inherently normative. Is there a reason not to include 'deontic declaration performer role' (IAO_0021022), 'document act template creator role' (IAO_0021021), and 'document act performer role' (IAO_0021031), and 'owner role' (IAO_0020027; child of OMRSE 'role in human social processes')

I take it as _paradigmatic_ that 'deontic declaration performer role' and 'owner role' are deontic roles.

**Proposal: all these terms belong under 'deontic role' (IAO_0021008) or a child-class of deontic role**

**Comment by jonathanvajda** [comment]
I made a poor word choice to use 'force'. I didn't meant to limit the class to legal contexts or enforcement of sanctions. The context of normative expectations is what I had in mind, and that the social world changes by social acts _according to_ these norms.
Could you give an example of one of these roles (or a realization of one of these roles) that isn't deontic? (E.g., if being an owner is not deontic, ...how?)

**Comment by jonathanvajda** [comment]
Permissions are included, yes, but also this other category: having the authority to do something (especially the authority to create, modify, or revoke deontic roles), as in the deontic power role. It seems that 'deontic declaration performer role' and 'document act performer role' are subclasses of 'deontic power role', and thus _a fortiori_ they are deontic roles.

One might worry that if permissions are also deontic roles, then if liberty is the natural state (Hobbesian state of nature, Crusoe's island, etc.) then there are deontic roles independent of the "normative expectations of others." This doesn't need to follow. Second, one might worry that if permissions are deontic roles, but there's an abundance of them that we would never need to talk about or represent ontologically (e.g., my permission to write in my notebook; 'this act of writing in this notebook is a realization of that deontic role'). This is true but not a problem.

Being an owner involves not only potentially possessing, using, and consuming the owned object, but also having the power to alter the social world regarding those who may possess, use, or consume it. The owner role may have non-deontic roles as parts. But an owner who is without power to change the status of permissions, obligations, etc. regarding an object is hardly an owner of that object by my estimation. (This even answers to a Marxist intuition that the abolition of private ownership is the abolition of the **power** or **right** that inheres in _only one_ individual rather than collectively.) So, I'm not sure what advantage there is for not bringing this central characteristic of ownership into our representation.

Suppose you'd say "An owner role is not a deontic role, but has a deontic role like 'deontic power role' as a role-part, necessarily." Creative solution, but it if is necessarily attached why would this not be a deontic role, then? And why would it be preferable to asserting that it is a deontic role?

Perhaps it is beneficial to make the following proposals piecemeal instead of my prior proposal, so we can all sort out each proposal Yea or Nay. I am more interested in getting some changes than all of the changes.

Proposal 1
I motion that we **assert 'deontic declaration performer role' (IAO_0021022) as a subclass of 'deontic role' (IAO_0021008) or a child-class of deontic role**

Proposal 2
I motion that we **assert 'document act performer role' (IAO_0021031) as a subclass of 'deontic role' (IAO_0021008) or a child-class of deontic role**

Proposal 3
I motion that we **assert 'owner role' (IAO_0020027; child of OMRSE 'role in human social processes') as a subclass of 'deontic role' (IAO_0021008) or a child-class of deontic role**

**Comment by mbrochhausen** [comment]
I looked at the matter and I still want to wait and see what Neil says, but I agree with Jonathan on  'deontic declaration performer role' (IAO_0021022), 'document act template creator role' (IAO_0021021), and 'document act performer role' (IAO_0021031). I am not certain regarding owner role, because it isn't formulated according to Hohfeldian terms. I think I might have thought about obsoleting it due to its current definition.

**Comment by neilotte** [comment]
When we defined 'deontic role', I believe our intent was to be sufficiently broad so as to allow for a variety of different deontic role subclasses. I think a criticism here would be that we may have cast our net too broadly, since--as @jonathanvajda rightly points out--many of the roles here seem to qualify. We might press the point further and wonder what subclasses of 'role' that have a social context would not also qualify as deontic roles on the present definition of deontic role in d-acts. I believe we've thought of such roles (e.g. the role of judge) as having deontic roles as parts, but that doesn't feature in the definition of the deontic role. 

These considerations lead me to think that rather than take @jonathanvajda up on his proposals now, that instead the definition of deontic role should be further restricted so as to exclude social roles that are subject to norms while retaining the present subclasses. Otherwise, deontic role is apt to swallow other social roles, like those of teacher and basketball coach, which was never the intent. 

I don't follow @eliasweatherfield 's claim that there are no normative expectations concerning the behavior of the bearer of a power. Those with a power role do seem to me subject to normative expectations on the appropriateness of the exercise of their power, as well as that it ought to be exercised. Perhaps I'm misunderstanding. 

As an aside, I tried to sketch the Hohfeldian System in OWL without presupposing a BFO/Dacts interpretation of the underlying ontology [here](https://github.com/neilotte/moralpsych/blob/master/rights-ontology.ttl), with the thought that if we worked out the logic in its own terms, we could later port it into dacts. Comments welcome on that file as well.

**Comment by jonathanvajda** [comment]
I agree with @neilotte to hold off on some of the proposals I was making.

I retract my request for these:
- Deontic Document Template Creator Role is not a deontic role.
- Owner Role is not a deontic role. Instances thereof might have deontic roles as parts, but the owner role is not a type of deontic role.

I still request change for this:
- Deontic Declaration Performer Role

**Comment by jonathanvajda** [comment]
Would the change I have requested for 'deontic declaration performer role' be a breaking change / require deprecation? I don't know the deprecation policy since what I propose does not break any inferences I know of (it's still a role, it just put into a subclass of role)

Apart from that, I don't have any competency questions that would be answered or required by this change. It just seems more accurate.
